python/py-absl/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-absl
+version             0.1.11
+platforms           darwin
+supported_archs     noarch
+license             Apache-2.0
+maintainers         {@emcrisostomo gmail.com:enrico.m.crisostomo} openmaintainer
+
+description         Abseil Python Common Libraries
+long_description    ${description}
+
+homepage            https://github.com/abseil/abseil-py
+master_sites        pypi:a/absl-py
+distname            absl-py-${version}
+
+checksums           md5    6abc83a2b86edd95f0f82447e4ad7088 \
+                    rmd160 80e4006b3a16c81bcb46fc18a71d6f58da889fb3 \
+                    sha256 ba6dc5510108fdfcf7176f8b38a5cc2482f2e902eb7211cbe6666998ea58daf9 \
+                    size   80018
+
+python.versions     27 34 35 36
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+        port:py${python.version}-setuptools \
+        port:py${python.version}-setuptools_scm
+
+    post-extract  {
+        system "chmod +r ${worksrcpath}/PKG-INFO"
+        system "chmod +r ${worksrcpath}/absl_py.egg-info/PKG-INFO"
```

Comment:
If you want you may consider Rainer's comments: native Tcl use is probably more correct (but this works as well).

I guess that an alternative solution would be to fetch from GitHub. I'm pretty sure you would not be facing the problem in that case :)